pkg/plugin/plugin.go

```diff
@@ -291,6 +303,13 @@ func (p *plugin) createOrUpdateExt(ctx context.Context, cs *api.OpenShiftManaged
 		cs.Properties.NetworkProfile.InternalLoadBalancerFrontendIPID = resourceid.ResourceID(cs.Properties.AzProfile.SubscriptionID, cs.Properties.AzProfile.ResourceGroup, "Microsoft.Network/loadBalancers", armconst.IlbAPIServerName+"/frontendIPConfigurations/"+armconst.IlbAPIServerFrontendConfigurationName)
 	}
 
+	if !isUpdate || cs.Properties.NetworkProfile.AllowVnetChangesToCauseNodeRotation {
```

Comment:
Do we want to override `AllowVnetChangesToCauseNodeRotation` to false at some point to ensure it is a transient flag and not written to the database?